Polyfills/Window/Source/TimeoutDispatcher.cpp

```diff
@@ -0,0 +1,150 @@
+#include "TimeoutDispatcher.h"
+
+#include <assert.h>
```

Comment:
https://stackoverflow.com/questions/43011737/is-it-better-to-include-cassert-or-assert-h
```suggestion
#include <cassert>
```